intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeature.kt

```diff
@@ -17,7 +17,8 @@ data class KtorFeature(
   override val pluginVersion: String? = null,
   override val pluginId: String? = null,
   val requiredFeatures: List<String> = emptyList(), // Feature IDs.
-  val documentation: KtorFeatureDocumentation? = null
+  val documentation: KtorFeatureDocumentation? = null,
+  val fullDescriptorJson: String = ""
```

Comment:
missing validation

Reply:
It is being created in one single place and the field gets set there. See `KtorFeaturePluginManager:110`. So it is just a string representation of the json that is already validated by that time